docs/source/intro_to_qc.ipynb

```diff
@@ -816,7 +816,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Before measurement: Bell state =  [ 0.70710678+0.j  0.00000000+0.j  0.00000000+0.j  0.70710678+0.j]\n",
+      "Before measurement: H|0> =  [ 0.70710678+0.j  0.00000000+0.j  0.00000000+0.j  0.70710678+0.j]\n",
```

Comment:
I think this reverts a previous PR (?) Can you remove from this pull request

Reply:
@mpharrigan ugh idk why that keeps happening, will do